src/wallet/wallet.cpp

```diff
@@ -4387,6 +4386,11 @@ bool CWallet::CreateZerocoinToSigmaRemintModel(string &stringError, int version,
     // currently we don't support zerocoin mints v1
     assert(version == ZEROCOIN_TX_VERSION_2);
 
+    if (!znodeSync.IsBlockchainSynced()) {
```

Comment:
I think we don't need this due to we already have below check. For me it also look weird too that wallet depend on znode.

Reply:
Agree

Reply:
this is not really a dependency on znode. This is a ready-to-use function that returns true if the blockchain is synced. There isn't another one.
If we don't use something like this the client can create remint while syncing/reindexing and the current block height is within the limits

Reply:
I see.